### ISSUE TITLE
Docker-compose up import fails

### ISSUE THREAD

**Issue by KevKo1990** [gravitystorm/openstreetmap-carto#4275]
### Expected behavior
I followed the description on how to setup docker from here: https://github.com/gravitystorm/openstreetmap-carto/blob/master/DOCKER.md

On my Windows 10 machine, I have docker running. In addition, I downloaded from geofabrik the `greece-latest.osm.pbf` file, placed it in the root folder of openstreetmap-carto and renamed it to `data.osm.pbf`.

Next, I started `docker-compose up import`. My expectation was that the docker-compose process runs without a problem.

### Actual behavior
`docker-compose up import` should actually import a local `data.osm.pbf`  file, however, stops too early and fails to complete the import.

The docker container with the carto_db stops/shutoffs with the following error message:

```
/usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/tune-postgis.sh
/docker-entrypoint-initdb.d/tune-postgis.sh: 6: /docker-entrypoint-initdb.d/tune-postgis.sh: Bad substitution
```

### Links and screenshots illustrating the problem
The shell output of `docker-compose up import` is:
```
C:\...\openstreetmap-carto>docker-compose up import
Creating network "openstreetmap-carto_default" with the default driver
Creating openstreetmap-carto_db_1 ... done                                                                              Creating openstreetmap-carto_import_1 ... done                                                                          Attaching to openstreetmap-carto_import_1
import_1   | Waiting for PostgreSQL to be running
import_1   | Timeout while waiting for PostgreSQL to be running
import_1   | psql: could not translate host name "db" to address: No address associated with hostname
import_1   | createdb: could not connect to database template1: could not translate host name "db" to address: No address associated with hostname
import_1   | osm2pgsql version 1.2.2 (64 bit id space)
import_1   |
import_1   | Allocating memory for dense node cache
import_1   | Allocating dense node cache in one big chunk
import_1   | Allocating memory for sparse node cache
import_1   | Sharing dense sparse
import_1   | Node-cache: cache=512MB, maxblocks=8192*65536, allocation method=11
import_1   | Mid: pgsql, cache=512
import_1   | Connection to database failed: could not translate host name "db" to address: No address associated with hostname
import_1   |
import_1   | Error occurred, cleaning up
import_1   | DB writer thread failed due to ERROR: Connection to database failed: could not translate host name "db" to address: No address associated with hostname
import_1   |
import_1   |
import_1   | /usr/bin/env: 'python3\r': No such file or directory
openstreetmap-carto_import_1 exited with code 127
```

The logs of the db docker container in detail are:
```
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
****************************************************
WARNING: No password has been set for the database.
         This will allow anyone with access to the
         Postgres port to access your database. In
         Docker's default configuration, this is
         effectively any other container on the same
         system.

         Use "-e POSTGRES_PASSWORD=password" to set
         it in "docker run".
****************************************************
waiting for server to start....2021-01-03 13:21:47.119 UTC [45] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2021-01-03 13:21:47.133 UTC [46] LOG:  database system was shut down at 2021-01-03 13:21:46 UTC
2021-01-03 13:21:47.137 UTC [45] LOG:  database system is ready to accept connections
 done
server started

/usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/postgis.sh
CREATE DATABASE
UPDATE 1
Loading PostGIS extensions into template_postgis
CREATE EXTENSION
CREATE EXTENSION
CREATE EXTENSION
CREATE EXTENSION
Loading PostGIS extensions into postgres
CREATE EXTENSION
CREATE EXTENSION
CREATE EXTENSION
CREATE EXTENSION

/usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/tune-postgis.sh
/docker-entrypoint-initdb.d/tune-postgis.sh: 6: /docker-entrypoint-initdb.d/tune-postgis.sh: Bad substitution
```

**Comment by KevKo1990** [comment]
I managed to skip the error by removing these two lines

```
"${psql[@]}" -c "ALTER SYSTEM SET work_mem='${PG_WORK_MEM:-16MB}';"
"${psql[@]}" -c "ALTER SYSTEM SET maintenance_work_mem='${PG_MAINTENANCE_WORK_MEM:-256MB}';"
```

in this file https://github.com/gravitystorm/openstreetmap-carto/blob/master/scripts/tune-postgis.sh

**Comment by ZeLonewolf** [comment]
I think this issue was fixed by #4330, can someone confirm whether this is still a problem?  I am currently running the docker commands on Windows 10 with no issues.

**Comment by pnorman** [comment]
Windows + `/usr/bin/env: 'python3\r': No such file or directory` is consistent with what #4330 fixed
